client/js/render.js

```diff
@@ -83,7 +83,7 @@ function buildChatMessage(data) {
 		text.html(templates.actions[type](data.msg));
 	}
 
-	if ((type === "message" || type === "action") && chan.hasClass("channel")) {
+	if ((type === "message" || type === "action" || type === "notice") && chan.hasClass("channel")) {
```

Comment:
Why did you need this addition? Sorry, not questioning, I'm just not seeing it 😅.